config/config.go

```diff
@@ -278,6 +115,63 @@ func createConfigFile(configFilePath string) {
 }
 
 func writeConfigFile(configFilePath string) error {
+	content := `# Configuration file for Dicedb
```

Comment:
I'm wondering if this should be a part of the code or if it would be better to store these in an actual file

Reply:
It is intended that the file be written to the default location, as I planned.